the_wizz/pair_maker.py

```diff
@@ -149,55 +150,6 @@ def run(self, reference_catalog, unknown_catalog, use_unkn_weights=False):
 
         return pd.DataFrame(output_data)
 
-    def _convert_radec_to_xyz(self, ras, decs):
-        """Convert RA/DEC positions to points on the unit sphere.
-
-        Parameters
-        ----------
-        ras : `numpy.ndarray`, (N,)
-            Right assertion coordinate in radians
-        decs : `numpy.ndarray`, (N,)
-            Declination coordinate in radians
-
-        Returns
-        -------
-        vectors : `numpy.ndarray`, (N, 3)
-            Array of points on the unit sphere.
-        """
-        vectors = np.empty((len(ras), 3))
-
-        vectors[:, 2] = np.sin(decs)
-
-        sintheta = np.cos(decs)
-        vectors[:, 0] = np.cos(ras) * sintheta
-        vectors[:, 1] = np.sin(ras) * sintheta
-
-        return vectors
-
-    def _query_tree(self, ref_vect, unkn_tree, dist):
```

Comment:
Add this back in along with the unittest to capture the conversion between cosmological distance and angle/euclidean distance.